Installation/src/CMakeLists.txt

```diff
@@ -80,6 +80,9 @@ function (collect_cgal_library LIBRARY_NAME ADDITIONAL_FILES)
     install(EXPORT ${LIBRARY_NAME}Exports
       DESTINATION "${CGAL_INSTALL_CMAKE_DIR}"
       NAMESPACE CGAL::)
+  else()
+    # Create an empty file (install mode) in order to be able to load it later
+    file(WRITE "${CMAKE_BINARY_DIR}/${LIBRARY_NAME}Exports.cmake" "")
```

Comment:
Instead of creating an empty file, I suggest that, when `CGAL_HEADER_ONLY` is `TRUE`, then the export file is not included!